javascript-view-swapping/main.js

```diff
@@ -0,0 +1,18 @@
+const $container = document.querySelector('.tab-container');
+const $tab = document.querySelectorAll('.tab');
+const $view = document.querySelectorAll('.view');
+
+$container.addEventListener('click', () => {
+     if (event.target.matches('.tab')) {
+      for (let i = 0; i < $tab.length; i++) {
+        $tab[i].className = 'tab'
+        event.target.className = 'tab active';
```

Comment:
You should have an `if` statement here comparing `event.target` to the current tab in the loop